apps/scan/backend/test/helpers/custom_helpers.ts

```diff
@@ -209,6 +209,10 @@ export function simulateScan(
   });
 }
 
+export async function waitForContinuousExportToUsbDrive(): Promise<void> {
```

Comment:
This operation happens in the background after a ballot is accepted, and this is the simplest way to ensure that it's able to finish in relevant tests. Ending a test before continuous export finishes can result in errors due to test directories getting cleaned up while they're still being written to.

Reply:
Ah yes, good call re a comment! https://github.com/votingworks/vxsuite/pull/4002/commits/5362fff7f48d2468193720be07d5828a67919928